day2_configuration.example.yaml

```diff
@@ -21,3 +21,8 @@ configurators:
         cluster_domain: <ocp.customer.domain.com>
         ca_pem_file_path: </path/to/ca.pem>
         ca_key_file_path: </path/to/ca.key>
+
+    nodes:
+        cluster_domain: <ocp.customer.domain.com>
+        firmware_files_dir: <firmware-local-dir>
```

Comment:
please add an explanation what should be placed there

Reply:
done